app/Presenters/ActionlogPresenter.php

```diff
@@ -38,22 +38,63 @@ public function item()
 
     public function icon()
     {
-        $itemicon = 'fas fa-paperclip';
-
-        if ($this->itemType() == 'asset') {
-            return 'fas fa-barcode';
-        } elseif ($this->itemType() == 'accessory') {
-            return 'far fa-keyboard';
-        } elseif ($this->itemType() == 'consumable') {
-            return 'fas fa-tint';
-        } elseif ($this->itemType() == 'license') {
-            return 'far fa-save';
-        } elseif ($this->itemType() == 'component') {
-            return 'far fa-hdd';
-        } elseif ($this->itemType() == 'user') {
-            return 'fa-solid fa-people-arrows';
+        
+        // User related icons
+        if ($this->itemType() == 'user') {
+
+            if ($this->actionType()=='create new') {
+                return 'fa-solid fa-user-plus';
+            }
+
+            if ($this->actionType()=='merged') {
+                return 'fa-solid fa-people-arrows';
+            }
+
+            if ($this->actionType()=='delete') {
+                return 'fa-solid fa-user-minus';
+            }
+
+            if ($this->actionType()=='delete') {
+                return 'fa-solid fa-user-minus';
+            }
+
+            if ($this->actionType()=='update') {
+                return 'fa-solid fa-user-pen';
+            }
+             return 'fa-solid fa-user';
         }
 
+        // Everything else
+        if ($this->actionType()=='create new') {
```

Comment:
Another candidate here for a `switch()` I'd argue.

Reply:
I had considered a switch, but since there is a nested if/else, it just seemed sort of clunky